packages/synapse-react-client/src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -658,26 +659,11 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
       </div>
       <div className="DatasetEditorTopBottomPanel">
         {hasChangedSinceLastSave && (
-          <Alert
-            dismissible={false}
-            show={true}
-            transition={false}
-            variant="warning"
-          >
-            <IconSvg
-              icon="warning"
-              sx={{
-                color: 'warning.main',
-                paddingRight: '0.2rem',
-              }}
-            />
```

Comment:
MUI warning alert includes the warning icon, so no longer need to include this -- 

main: 
<img width="1482" alt="main_DatasetItemsEditor" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/58079791-0e9d-4d49-bd36-ec308059d92a">

feature: 
<img width="1466" alt="feature_DatasetItemsEditor" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/d5d8c75b-7e65-4308-8a7d-d109d73e4966">